contents/handbook/small-teams/pipeline/objectives.mdx

```diff
@@ -1,3 +1,25 @@
+## Objectives: Q4 2023
+
+- Keep the lights on
+  - Increase isolation with Redis - Xavier Vello
+  - Roll out RDKafka - Brett Hoerner
+  - All events ingested through capture-rs - Xavier Vello
+  - Make GeoIP cheaper - Ted
+  - moving event properties follow-ups, cohorts, ... outside of ingestion hot path - Ted
+- Batch exports migrations - Tomás Farías Santana
+  - Redshift and maybe replicator depending on demands
+  - Add logs/metrics/alerting
+  - Higher frequency
+- Webhooks v2.1 - Tiina Turban
+  - Svix/delivery reliability
+  - Design and build a high level automation thing that webhooks/actions can use
+  - Behavioural cohorts for feature flags
+  - Alerting on events thresholds
```

Comment:
This is not understandable to the reader, so let's list out what it actually means instead (based on https://docs.google.com/document/d/1ePzrFs1O6pCgqzXfeLQcMqF8_8VxN0z4MB73aqyCoOc/edit#heading=h.5gwiik2lhtyx ), also not sure about linking the objective to Svix, as if that doesn't work out we'll just need to build something ourselves.
```suggestion
- Webhooks v2 - Tiina Turban
  - new delivery system:
  	- retries
 	- observability to both users and us for success rate, retry count, latency
 	- endpoint disabling (per user:destination)
 	- multiple webhook destinations
 	- UI
 - move webhooks, resthooks and onEvent apps to use the new system
```